content/tutorials/portenta-h7/por-ard-gs/content.md

```diff
@@ -84,23 +42,16 @@ This step is the same for both the classic IDE and the Pro IDE. Open the board m
 
 ![Also in the Pro IDE, a search for "mbed" reveals the core that needs to be installed to support Portenta H7.](assets/por_ard_gs_bm_core_pro_ide.png)
 
-
-## 4. Install the USB driver (Windows only)
-This step is only needed for Microsoft Windows users and only needs to be done once. To be able to recognise the Portenta H7 in windows you need to install a generic USB device driver. Zadig is an open source software that can be used for this. Install[ Zadig](https://zadig.akeo.ie/), then connect your portenta with the USB cable and assign WinUSB to Portenta H7 Bootloader as shown in the image below. To finish click on the Install Driver button.
-
-![Assigning WinUSB to Portenta H7 Bootloader.](assets/por_ard_gs_zadig_driver.png)
-
-
-## 5. Verify the USB connection  (Windows only)
+## 3. Verify the USB connection  (Windows only)
 In this step you will make sure that the drivers are installed correctly and Windows is able to detect the Portenta H7. To do so open the Windows Device manager and if everything is set up correctly you will be able to see your device listed under USB devices. Otherwise go back to step 4 and try to re-install the drivers.
```

Comment:
```suggestion
In this step you will check if Windows is able to detect the Portenta H7. To do so open the Windows Device manager and if everything is set up correctly you will be able to see your device listed under USB devices. Otherwise, try unplugging it and plugging it back in.
```